tests/command_worker_integration.rs

```diff
@@ -479,7 +475,7 @@ async fn test_annotation_stop() {
 
 async fn wait_for_event(sub: &mut async_nats::Subscriber, match_text: &str) {
     // Providers can take a bit to start if they are downloading
-    let mut interval = tokio::time::interval(std::time::Duration::from_secs(15));
+    let mut interval = tokio::time::interval(std::time::Duration::from_secs(30));
```

Comment:
I bumped this up to 30, because I noticed it was fail reasonably often with 15, but not once at 30 in my testing.

It's of course not very scientific, but I think having to wait a bit longer is a reasonable trade off to having to re-run the entire suite.